server/jetstream_api.go

```diff
@@ -817,7 +817,9 @@ func (a *Account) trackAPI() {
 		jsa.usage.api++
 		jsa.apiTotal++
 		jsa.sendClusterUsageUpdate()
+		js := jsa.js
 		jsa.mu.Unlock()
+		atomic.AddInt64(&js.apiTotal, 1)
```

Comment:
Since this is atomic op, you don't need to capture `js := jsa.js` and do this here outside of `jsa`'s lock. So I would move this to after the line `jsa.apiTotal++` (before sendClusterUsageUpdate())

Reply:
Fixed.